swagger.yml

```diff
@@ -561,15 +561,26 @@ definitions:
         $ref: '#/definitions/euro'
       kindergeld:
         $ref: '#/definitions/euro'
-      nebentaetigkeit:
-        $ref: '#/definitions/euro'
+      nebentaetigkeiten:
```

Comment:
Würde hier auch die Summe der Einkünfte aus Nebentätigkeit reichen oder wird das Datum benötigt? In der ME interessiert sich bisher niemand für das Datum. Daher würde ich das auch nicht weiter propagieren, dann könnte man es eventuell mal aus dem Frontend ausbauen.

Reply:
es wird explizit das Datum benötigt, wurde beim letzten Meeting durch die Alte Leipziger angefragt